src/resources/postcss/base/typography/_headings.pcss

```diff
@@ -61,9 +61,9 @@
 	.tribe-common-h4 {
 		color: var(--color-text-primary);
 		font-family: var(--font-family-sans-serif);
-		font-size: var(--font-size-5);
+		font-size: var(--font-size-7);
```

Comment:
we actually need to keep this at `--font-size-5` and `--line-height-1` for mobile, but below, add a media query for `--viewport-medium` with the desktop size (`--font-size-7`)